aptos-move/aptos-aggregator/src/delta_change_set.rs

```diff
@@ -295,6 +295,23 @@ impl DeltaChangeSet {
         }
     }
 
+    pub fn contains_key(&self, key: &StateKey) -> bool {
+        self.delta_change_set.contains_key(key)
+    }
+
+    pub fn get_write_op(
```

Comment:
I would argue we do not need this with extra expects, etc. Why not just get the delta?

```
// DeltaStateView
match self.delta_set.get(state_key) {
   Some(...) => ...
   None => match self.write_set.get(state_key) {
     // Old logic.
   }
}
```

See other comment

Reply:
Done.